scenarios/change-data-capture/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.9"
+services:
+  pravega:
+    image: fyang86/pravega:0.10.1-65df3fbd-set-published-ip
+    ports:
+      - "9090:9090"
+      - "12345:12345"
+    command: standalone
+
+  mysql:
+    image: thekingofcity/pravega-cdc-demo-mysql:0.1
+    ports:
+      - "3306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=dbz
+
+  datagen:
+    image: thekingofcity/pravega-cdc-demo-datagen:0.1
+    depends_on:
+      - mysql
+    environment:
+      - wait_for=45
+
+  debezium:
+    image: thekingofcity/pravega-cdc-demo-debezium:0.1
+    depends_on:
+      - pravega
+      - datagen
+    command: sh -c "
+      sleep 60 &&
```

Comment:
Instead of a static wait for, maybe having a health checker is better, like what is did in the Catalog demo PR.